.env.sample

```diff
@@ -24,20 +24,13 @@ RESPONSE_S_MAXAGE=0
 AUTH_JWT_SECRET=f023d3db-39dc-4ac9-87b2-b2be72e9162b
 AUTH_JWT_ALGORITHM=HS256
 
-
-#######################################
-# Single Tenant
-#######################################
-TENANT_ID=bjhaohmqunupljrqypxz
-
 #######################################
 # Multi Tenancy
 # uncomment MULTI_TENANT=true to enable multi tenancy
-# the [Single Tenant] configurations will be ignored
 #######################################
 # MULTI_TENANT=true
 DATABASE_MULTITENANT_URL=postgresql://postgres:postgres@127.0.0.1:5433/postgres
-REQUEST_X_FORWARDED_HOST_REGEXP=
+REQUEST_X_FORWARDED_HOST_REGEXP=^([a-z]{20}).local.(?:com|dev)$
```

Comment:
default value for multitenant x-forwarded-host: `{tenant-id}.local.dev` to make it easy to run Storage out of the box as multitenant